test/orderby.js

```diff
@@ -134,5 +134,89 @@ export default (test) => {
 		nestedFilterTest.only = testFilterOption.bind(null, test.only);
 
 		filterby(nestedFilterTest);
+
+		test('$orderby=Products(1) asc', [1], function (result) {
```

Comment:
I assume `$orderby=Products(1)/Orders/$count asc` would currently be unsupported which is fine, but is it something that technically should be (for later work)

Reply:
It seems that this & the alternate key works already as expected